.github/workflows/Windows.yml

```diff
@@ -118,7 +118,7 @@ jobs:
 
     - uses: actions/upload-artifact@v2
       with:
-        name: duckdb-binaries-windows
+        name: duckdb-binaries-windows-32bit
```

Comment:
This was causing a name collision, stopping 64bit java windows builds from being uploaded